docs/guides/SimpleStartupGuide.md

```diff
@@ -0,0 +1,396 @@
+| **curl**     | Allows the user to connect to services such as EdgeX. |Use curl to get transfer information either to or from this service. In the tutorial, use `curl` to communicate with the EdgeX API. The call will return a JSON object.|
+| **mplayer** |  used to view the video stream | |
+| **v4l-utils** | used to determine the video stream path of a usb camera | |
+
+>Table 1: Command Line Tools
+## Get the Source Code
+###  Download EdgeX Compose Repository
+
+1. Create a directory for the EdgeX compose repository:
+   ```bash
+   mkdir ~/edgex
+   ```
+
+2. Change into newly created directory:
+   ```bash
+   cd ~/edgex
+   ```
+
+3. Clone the EdgeX compose repository
+   ```bash
+   git clone https://github.com/edgexfoundry/edgex-compose.git
+   ```
+
+
+## Run the Service
+
+1. Navigate to the Edgex compose directory.
+
+   ```shell
+   cd ~/edgex/edgex-compose/compose-builder
+   ```
+
+2. Run EdgeX with the microservice:  
+  - For non secure mode
+    ```
+    make run ds-usb-camera no-secty
+    ```
+  - For secure mode 
+    ```
+    make run ds-usb-camera
+    ```
+
+## Verify Service and Device Profiles
+
+1. Check the status of the container:
+
+   ```bash 
+   docker ps
+   ```
+
+   The status column will indicate if the container is running and how long it has been up.
+
+   Example Output:
+
+   ```docker
+   CONTAINER ID   IMAGE                                         COMMAND                  CREATED       STATUS          PORTS                                                                                         NAMES
+   f0a1c646f324   edgexfoundry/device-usb-camera:0.0.0-dev                        "/docker-entrypoint.…"   26 hours ago   Up 20 hours   127.0.0.1:8554->8554/tcp, 127.0.0.1:59983->59983/tcp                         edgex-device-usb-camera                                                                   edgex-device-onvif-camera
+   ```
+
+2. Check that the device service is added to EdgeX:
+
+   ```bash
+   curl -s http://localhost:59881/api/v2/deviceservice/name/device-usb-camera | jq .
+   ```
+   
+   Successful:
+   ```json
+   {
+      "apiVersion": "v2",
+      "statusCode": 200,
+      "service": {
+         "created": 1658769423192,
+         "modified": 1658872893286,
+         "id": "04470def-7b5b-4362-9958-bc5ff9f54f1e",
+         "name": "device-usb-camera",
+         "baseAddress": "http://edgex-device-usb-camera:59983",
+         "adminState": "UNLOCKED"
+      }
+   }
+   ```
+   Unsuccessful:
+   ```json
+   {
+      "apiVersion": "v2",
+      "message": "fail to query device service by name device-usb-camera",
+      "statusCode": 404
+   }
+   ```
+## Adding Devices using REST API
+Devices can either be added to the service by defining them in a static configuration file, discovering devices dynamically, or with the REST API. For this example, the device will be added using the REST API.
```

Comment:
I don't think the simple guide should include this, as it will conflict with the auto-discovered device. Instead we should show the user how to query the devices that were added via discovery.